packages/jupyter/src/execute/provider.tsx

```diff
@@ -49,11 +50,11 @@ function useScopeNavigate({
       return;
     }
 
-    const computables: Computable[] = selectAll('container > embed', mdast).map((node: any) => {
+    const computables: Computable[] = selectAll('container > output', mdast).map((node: any) => {
       const { key, label, source } = node;
       const output = selectAll('output', node);
-      if (output.length === 0) console.error(`embed must have exactly one output ${key}`);
-      if (output.length > 1) console.warn(`embed has more than one output block ${key}}`);
+      if (output.length === 0) console.error(`container must have exactly one output ${key}`);
```

Comment:
We won't hit this any more, since we are only selecting containers with outputs now, right? Do we still want to error on other containers without outputs?

Reply:
ok seems like we need to consider both the `container > output` and `embed > output` cases here